### PR TITLE
migration/checkStave

### PR DESCRIPTION
**Pull request by rvilarl**
NoStave error simplified using checkStave

### CODE REVIEW

File: src/clefnote.ts [hunk 1]
```diff
@@ -73,17 +73,16 @@ export class ClefNote extends Note {
   }
 
   draw(): void {
-    if (!this.stave) throw new Vex.RERR('NoStave', "Can't draw without a stave.");
```
Comment (0xfe) [inline src/clefnote.ts:76]: Can you remove `Vex` from the `import`?

File: src/glyphnote.ts [hunk 2]
```diff
@@ -49,20 +49,17 @@ export class GlyphNote extends Note {
   }
 
   draw(): void {
-    if (!this.stave) {
-      throw new Vex.RERR('NoStave', 'No stave attached to this note.');
```
Comment (0xfe) [inline src/glyphnote.ts:53]: Can you remove `Vex` from the `import`?

File: src/keysignote.ts [hunk 3]
```diff
@@ -23,22 +23,15 @@ export class KeySigNote extends Note {
   }
 
   preFormat(): this {
-    if (!this.stave) {
-      throw new Vex.RERR('NoStave', 'No stave attached to this note.');
```
Comment (0xfe) [inline src/keysignote.ts:27]: Can you remove `Vex` from the `import`?

File: src/timesigglyph.ts [hunk 4]
```diff
@@ -57,8 +58,7 @@ export class TimeSignatureGlyph extends Glyph {
   }
 
   renderToStave(x: number): void {
-    if (!this.stave) throw new Vex.RERR('NoStave', "Can't render without a stave.");
```
Comment (0xfe) [inline src/timesigglyph.ts:60]: Can you remove `Vex` from the `import`?

**Review by 0xfe — COMMENTED** [review]
Thanks for this PR! Some minor comments.

File: src/clefnote.ts (hunk 1 above)
Comment (rvilarl) [reply src/clefnote.ts:76]: done

File: src/glyphnote.ts (hunk 2 above)
Comment (rvilarl) [reply src/glyphnote.ts:53]: done

File: src/keysignote.ts (hunk 3 above)
Comment (rvilarl) [reply src/keysignote.ts:27]: done

File: src/timesigglyph.ts (hunk 4 above)
Comment (rvilarl) [reply src/timesigglyph.ts:60]: done

**Comment by rvilarl** [comment]
Ready to merge!

**Comment by 0xfe** [comment]
Looks good. Thanks!

**Pull request merged**